.github/workflows/build.yaml

```diff
@@ -14,6 +14,18 @@ on:
     - release-*
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 7.0.x
+      - name: Lint
+        run: dotnet format --verify-no-changes
```

Comment:
I know people that hates style enforces. I personally rather have a tool dictate and reformat what the style should be than having to spend time on PRs discussing where brackets should be to improve legibility. And legibility and uniformity matters :)

Reply:
Yes, I'm happy to just have *a* style defined, and I've grown to care less and less about *which* style it happens to be.